hooks/run-relevant-tests.sh

```diff
@@ -0,0 +1,35 @@
+STAGED_FILES=$(git diff --cached --name-only --diff-filter=ACM | grep "js$")
+if [ "$STAGED_FILES" = "" ]; then
+    exit 0
+fi
+
+PASS=true
+
+for FILE in $STAGED_FILES
+do
+    yarn eslint --quiet "$FILE"
+
+    if [ "$?" == 0 ]; then
+        echo "\t\033[32mESLint Passed: $FILE\033[0m"
+    else
+        echo "\t\033[41mESLint Failed: $FILE\033[0m"
+        PASS=false
+    fi
+
+    yarn prettier "$FILE"
+
+    if [ "$?" == 0 ]; then
+        echo "\t\033[32mPrettier  Passed: $FILE\033[0m"
+    else
+        echo "\t\033[41mPrettier Failed: $FILE\033[0m"
+        PASS=false
+    fi
+done
```

Comment:
@samschaevitz this is running eslint on the whole application per file changed, prettier stage also prettifies all the app, either we change/remove yarn eslint/prettier scripts or we use the eslint/prettier executable directly, also i think we should use $STAGED_FILES in a single command instead of an iteration ? what do you think.